DataFormats/common/include/CommonDataFormat/InteractionRecord.h

```diff
@@ -197,6 +197,9 @@ struct InteractionRecord {
   {
     // bc self-addition operator, no check for orbit wrap
     auto l = toLong() + dbc;
+    if (l < 0) {
+      l += MaxGlobalBCs;
+    }
```

Comment:
this is not correct, please suppress this patch, I've opened a PR https://github.com/AliceO2Group/AliceO2/pull/10528 with a proper check of wrapping for all operations.

Reply:
PLease, see my comment in 10528.